lib/bike_brigade/importers/mailchimp_importer.ex

```diff
@@ -77,9 +78,20 @@ defmodule BikeBrigade.Importers.MailchimpImporter do
     Map.put(rider_attrs, :location_struct, %{address: "1 Front St"})
   end
 
+  def tag_invalid_location(rider) do
+    # TODO make this an idempotent Riders.add_tag
+    rider =
+      rider
+      |> Repo.preload(:tags)
+
+    tags = Enum.map(rider.tags, & &1.name) ++ ["invalid_location"]
```

Comment:
Do you think people will remember to remove the tag once they fix the address?

Reply:
I hope so! I added a note to the slack message